pkgs/desktops/mate/mate-tweak/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+, intltool
+, mate
+, libnotify
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+, glib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "mate-tweak";
+  version = "20.04.0";
+
+  src = fetchFromGitHub {
+    owner = "ubuntu-mate";
+    repo = pname;
+    rev = version;
+    sha256 = "14cvh5hf61s0z75skrpz6r0z27jd01z100v2cqih2ck0pxv0483v";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    intltool
+    python3Packages.distutils_extra
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    gtk3
+    libnotify
+    glib
+    mate.mate-applets
+    mate.mate-panel
+    mate.marco
+    mate.libmatekbd
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    distro
+    pygobject3
+    psutil
+    setproctitle
+  ];
+
+  strictDeps = false;
+
+  dontWrapGApps = true;
+
+  postPatch = ''
+    # mate-tweak hardcodes absolute paths everywhere. Nuke from orbit.
+    find . -type f -exec sed -i \
+      -e s,/usr/lib/mate-tweak,$out/lib/mate-tweak,g \
+      {} +
+  '';
```

Comment:
there's still a ton more hardcoded paths in this project.
I'm also seeing code here https://github.com/ubuntu-mate/mate-tweak/blob/2ccffffdff2b2f831cd05f787e37c0f297c1422f/util/mate-tweak-helper#L30, that will probably be entirely broken on nixos

Reply:
I've talked with @luc65r and the result was that it would be a lot of work to actually fix every usecase and that making it work on mate properly should be enough for shipping